mutt/file.h

```diff
@@ -144,6 +145,7 @@ void        buf_quote_filename(struct Buffer *buf, const char *filename, bool ad
 void        buf_file_expand_fmt_quote(struct Buffer *dest, const char *fmt, const char *src);
 
 #define mutt_file_fopen(PATH, MODE) mutt_file_fopen_full(PATH, MODE, __FILE__, __LINE__, __func__)
+#define mutt_file_fopen_masked(PATH, MODE) mutt_file_fopen_full_masked(PATH, MODE, __FILE__, __LINE__, __func__)
```

Comment:
If we really need to get the mode from `NeoMutt`,
then we might need to move the wrapper to `core/`